algorithms.py

```diff
@@ -33,6 +33,7 @@
 from dask.distributed import Client
 from dask_cuda import LocalCUDACluster
 import xgboost as xgb
+import cudf
```

Comment:
Is cudf used? If not let's remove the import. If someone wants to run the scripts outside of the docker container, dependencies like these make it harder.

Reply:
I am importing cudf here because it is a hard dependency for the cuml rf module.

Reply:
In general, cudf is a hard dependency of cuml, according to this Conda recipe: https://github.com/rapidsai/cuml/blob/3ea117d3fc8ab469210a830db5a1445fad0186fd/conda/recipes/libcuml/meta.yaml#L35